src/timer.rs

```diff
@@ -2,25 +2,30 @@ use core::mem;
 use core::cell::Cell;
 use syscalls::{self, command, yieldk_for};
 
+const DRIVER_NUMBER: u32 = 0;
+const GET_CLOCK_FREQUENCY: u32 = 1;
+const SET_ALARM_NOTIFICATION: u32 = 4;
+const GET_CLOCK_VALUE: u32 = 2;
+
 pub unsafe fn subscribe(cb: extern fn(usize, usize, usize, usize), ud: usize) {
     syscalls::subscribe(3, 0, cb, ud);
 }
 
 pub fn start_oneshot(ms: u32) {
```

Comment:
The name for this function should change now that it has different functionality (sorry it's confusing, there used to be oneshot functionality in the kernel, but we removed it and just left the set_alarm functionality)